src/retasc/validator/models.py

```diff
@@ -0,0 +1,44 @@
+from pydantic import BaseModel, Field
+
+
+class JiraIssue(BaseModel):
+    """Represents a Jira issue, which can have subtasks."""
+
+    template: str = Field(..., description="The template string for the jira issue.")
```

Comment:
Interesting, TIL that the `Ellipsis` (`...`) indicates that the field is required. Unfortunately, it is more confusing than useful (I had to look it up).

Can you remove it? AFAIK, setting `default` makes the field optional.

Reply:
Changed